src/constants.js

```diff
@@ -310,6 +310,7 @@ export const ORIGIN_PREVIEW: 'preview' = 'preview';
 export const ORIGIN_PREVIEW_SIDEBAR: 'preview_sidebar' = 'preview_sidebar';
 export const ORIGIN_CONTENT_EXPLORER: 'content_explorer' = 'content_explorer';
 export const ORIGIN_OPEN_WITH: 'open_with' = 'open_with';
+export const ORIGIN_ANNOTATION_THREAD: 'annotation_thread' = 'annotation_thread';
```

Comment:
Let's revert this change. These are intended for use by top-level Elements like ContentSidebar.

Reply:
I believe there is some misunderstanding - AnnotationThread is meant to be a top level component. The plan for AnnotationThread is to exist independently form Sidebar, wrap AnnotationActivity with all its functionalities and possibility to create new Annotation as well (will be added in next PRs)

Reply:
Discussed offline, but to capture here: We can have top-level components/features that aren't top-level UI Elements. It looks like this new component is the former rather than the latter.